SeaPublicWebsite/ExternalServices/Models/Epb/EpbEpcDto.cs

```diff
@@ -522,81 +521,203 @@ public Epc Parse()
         };
     }
 
-    private HeatingType? ParseHeatingType()
+    private EpcHeatingType? ParseHeatingType()
     {
-        // Gas boiler check
-        // 20 - mains gas (community)
-        // 26 - mains gas (not community)
-        if (MainFuelType.Equals("20") ||
-            MainFuelType.Equals("26") ||
-            MainFuelType.Contains("mains gas", StringComparison.OrdinalIgnoreCase))
+        if (MainFuelType is null)
         {
-            return HeatingType.GasBoiler;
+            return null;
         }
         
-        // Oil boiler check
-        // 22 - oil (community)
-        // 28 - oil (not community)
-        if (MainFuelType.Equals("22") ||
-            MainFuelType.Equals("28") ||
-            MainFuelType.Contains("oil", StringComparison.OrdinalIgnoreCase))
+        // Anything ending in '(community)'
```

Comment:
Is this definitely correct? There are a lot of changes below in how we parse the EPC values. If this is correct where is it documented?